cms/static/sass/elements/_modal-window.scss

```diff
@@ -579,28 +579,33 @@
   .wrapper-modal-window-bulkpublish-subsection,
   .wrapper-modal-window-bulkpublish-unit,
   .course-outline-modal {
-    .exam-time-list-fields,
-    .exam-review-rules-list-fields {
-      margin: 0 0 ($baseline/2) 0;
+    .no-descriptive-text {
```

Comment:
Note to reviewers: this is for this case, where there is a radio button with no descriptive text underneath it. It is needed now because we are no longer using the ul/li nesting.

![image](https://cloud.githubusercontent.com/assets/484484/26125424/1e5bad24-3a50-11e7-8d6f-60ce39ed8535.png)